### PR TITLE
MAINT: Remove np.pkgload, which seems to be unusable anyway

### PR DESCRIPTION
**Pull request by eric-wieser**
Attempts to use this on either cpython27 or cpython35, in either a shell or a script, gives:

    NameError: name '__path__' is not defined

### CODE REVIEW

File: numpy/_import_tools.py [hunk 1]
```diff
@@ -1,351 +0,0 @@
...
-             verbose=-1 will suspend also warnings.
-        force= : bool
-             when True, force reloading loaded packages [default: False].
-        postpone= : bool
-             when True, don't load packages [default: False]
-
-        """
-        # 2014-10-29, 1.10
-        warnings.warn('pkgload and PackageLoader are obsolete '
-                'and will be removed in a future version of numpy',
```
Comment (eric-wieser) [inline numpy/_import_tools.py]: Didn't even spot this. 3ish years seems like plenty of time to wait, especially since it seems we broke it anyway

**Comment by charris** [comment]
Needs a release note, otherwise LGTM.

**Comment by eric-wieser** [comment]
Updated

**Comment by charris** [comment]
Thanks Eric.

**Pull request merged**